sonar-db/src/main/java/org/sonar/db/permission/PermissionTemplateDao.java

```diff
@@ -296,20 +296,28 @@ public void insertGroupPermission(DbSession session, Long templateId, @Nullable
     session.commit();
   }
 
+  /**
+   * @deprecated since 5.2 use the method with DbSession
```

Comment:
rather than "use the method with DbSession", you could use a javadoc `{@link #...}`, more accurate and IDE can refactor it and display a warning if link becomes broken at some point